### PR TITLE
4 base template

### PR DESCRIPTION
**Pull request by SNESChalmers34**
Made a new header/footer in base.html and base.css for the base template

### CODE REVIEW

**Comment by ehenry2172** [comment]
Looks good except footer is in the wrong place when the template is rendered...see for the following working screenshot example.

tw_screenshot

File: twittosphere/templates/base.html [hunk 1]
```diff
@@ -16,27 +16,46 @@
...
+                <li class="active"><a href="#">Home</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li> 
+                <li><a href="#">Search</a></li> 
             </ul>
-        </nav>
+        </div>
+    </nav>
+
+    <div class="content">
```
Comment (ehenry2172) [inline twittosphere/templates/base.html]: You put a div for content here, but the content block is on lines 49-51...

File: twittosphere/templates/base.html [hunk 2]
```diff
@@ -16,27 +16,46 @@
...
+    <!-- Footer -->
+    <footer>
+        <nav class="navbar navbar-inverse navbar-bottom">
+            <div class="container-fluid">
+                <p id="copyright">© Mark Sweet 2016</p>
+            </div>
+        </nav>
+    </footer>
     <!-- Content block -->
     {% block content %}
```
Comment (ehenry2172) [inline twittosphere/templates/base.html]: Move the content block inside your content div above.

File: twittosphere/static/css/base.css [hunk 3]
```diff
@@ -1,3 +1,29 @@
+* {
+	margin:0;
+	-webkit-border-radius: 0 !important;
+    -moz-border-radius: 0 !important;
```
Comment (ehenry2172) [inline twittosphere/static/css/base.css]: fix indent.

File: twittosphere/static/css/base.css [hunk 4]
```diff
@@ -1,3 +1,29 @@
...
+
+.content{
+    flex-grow: 1;
+}
+
+.navbar{
+    margin-bottom: 0;
+}
+
+#copyright{
```
Comment (ehenry2172) [inline twittosphere/static/css/base.css]: it looks better if you have a space after the class name and the brace...see line 27 for example.

Comment (SNESChalmers34) [inline twittosphere/static/css/base.css]: Are you gonna be on League tonight?

**Comment by ehenry2172** [comment]
Looks good. merging it now.

**Pull request merged**